### PR TITLE
docs: Update Manufacturing module

### PR DESCRIPTION
**Pull request by michellealva**
some grammatical corrections

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md [hunk 1]
```diff
@@ -48,6 +48,6 @@ with the Project module.
 You can track work-in-progress by creating work-in-progress Warehouses.
 
 ERPNext will help you track material movement by automatically creating Stock
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md]: unsure about this change, the meaning changes... need to confirm which one is right

File: erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md [hunk 2]
```diff
@@ -48,6 +48,6 @@ with the Project module.
...
-Entries from your Work Orders by building from Bill of Materials.
+Entries from your Work Orders by creating Bill of Materials.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md]: note to self: unsure about this change, the meaning changes... need to confirm which one is right

Comment (michellealva) [reply erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md]: Ok stick to the first one... I confirmed...

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md]: ok, so make the change

**Pull request merged**